_docs/_latest/setup/kubernetes.md

```diff
@@ -0,0 +1,124 @@
+installer](https://github.com/terraform-google-modules/terraform-google-forseti).
+This is a quick way to install the service accounts, Cloud SQL and GCS buckets.
+The VM's may be deleted or shutdown after the install as they will be replaced
+by the k8s solution.
+
+Alternatively follow the [manual setup instructions]({% link _docs/latest/setup/manual.md %})
+for service accounts, Cloud SQL and GCS buckets.
+
+## Build Docker Image
+
+There are various ways to build the docker image, using git clone or
+a personal fork.
+
+Note: [cloudbuild.yaml](https://github.com/GoogleCloudPlatform/forseti-security/blob/docker-poc/install/docker/cloudbuild.yaml)
+base image cache step will fail first time as their is no base image to cache,
+comment this out manually if it's not yet commented out in the codebase.
+
+BUILD IMAGE (git clone example)
+```
+Clone the forseti security repo to the Cloud Shell VM in your project
+cd to forseti-security directory
+git checkout dev (or other branch)
+gcloud builds submit --config install/docker/cloudbuild.yaml .
+(Dont forget the dot at the end)
+```
+Ref. https://cloud.google.com/cloud-build/docs/quickstart-docker#build_using_a_build_config_file
+
+BUILD IMAGE (Forked repo trigger example)
+```
+Create a fork of the forseti security repository in github.
+Create a Cloud Build Trigger to automatically build when changes pushed to your forked repo
+Specify the branch (e.g. dev)
+Specify the cloudbuild.yaml build config file
+```
+
+Example:
+{% responsive_image path: images/docs/setup/k8s_cloud_build.png alt: "k8s cloud build" %}
+
+## Deploy to Kubernetes
+
+1. Open a cloud shell VM in your project.
+1. Upload the server and client service account keys to Cloud Shell.
+Note the file locations for later.
+1. Clone your forked repo and switch to the `dev` branch.
+1. Modify [forseti-security/install/scripts/k8s_setup_forseti.sh](https://github.com/GoogleCloudPlatform/forseti-security/blob/docker-poc/install/scripts/k8s_setup_forseti.sh)
+* setting the environment variables
+* paths to your service account keys
+* choose the architecture, either k8s CronJob or long running server
+1. `cd to ../forseti-security/install/scripts/` and run the `k8s_setup_forseti.sh` script.
+
+NOTE: In client config file in GCS bucket, change IP of forseti-server
+to match the Cluster Ip of Forseti Service
+
+Ref. https://github.com/GoogleCloudPlatform/forseti-security/blob/docker-poc/install/scripts/k8s_setup_forseti.sh
+
+## Verify
+
+Monitor the deployment in the GKE web console. Allow approximately 2 minutes
+for the cluster to spin up and another 30 seconds for the pods to become active.
+
+Long running server example:
+{% responsive_image path: images/docs/setup/k8s_long_running_server.png alt: "k8s long running server" %}
+
+Run `kubectl get pods` to get pod ids
+Connect: `kubectl exec -it <CLIENT_POD_ID> -- /bin/bash`
+
+After connecting to client, source the environment variables before
```

Comment:
Pls remove this paragraph. Vars are automatically sourced now.

Reply:
Done.